.github/workflows/main.yaml

```diff
@@ -0,0 +1,33 @@
+name: Ruby
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Because it's the default and simplest configuration for Github Action. In any case your host will run on ubuntu. You can then run a docker image, it will run in Docker but your host will still run Ubuntu.

Reply:
Yes.